text_machina/src/extractors/__init__.py

```diff
@@ -8,6 +8,8 @@
 from .combined import Combined
 from .dummy import Dummy
 from .entity_list import EntityList
+from .gap_sentence import GapSentence
```

Comment:
Minor comment: can we name these "SentenceGap" and "WordGap"?

Reply:
sure! done.